Dockerfile

```diff
@@ -544,6 +544,7 @@ RUN --mount=type=cache,sharing=locked,id=moby-dev-aptlib,target=/var/lib/apt \
             libprotobuf-c1 \
             libyajl2 \
             net-tools \
+            netcat-openbsd \
```

Comment:
Is this only for our tests, or is this a new dependency we should have installed (and to be added as "requires" in our packages?)

I can't find a direct reference to this in this PR; was it accidentally left behind (for local debugging?)

Reply:
That's only for testing. Here's where it's used: https://github.com/moby/moby/pull/48721/files#diff-4fd76a7e4270f16a03d5160517ee2454558f127af10e95c55451963b17429414R817